src/views/EditorHome.vue

```diff
@@ -546,6 +547,8 @@ export default defineComponent({
         store.state.acceptTerms !== "Accepted";
 
       isCompletedInitialStartup.value = true;
+      // Engine started, close splash
+      store.dispatch("ON_CLOSE_SPLASH");
```

Comment:
I noticed in reviewing the code that if a new character is found, a character selection dialog appears.
https://github.com/VOICEVOX/voicevox/blob/c1ae436cb643c78b51653eeb5b63d194f132f30a/src/views/EditorHome.vue#L511-L513

So I would like to close the splash screen before the engine startup completion.
How about closing the splash screen just after this, when one `AudioCell` is created?
https://github.com/VOICEVOX/voicevox/blob/c1ae436cb643c78b51653eeb5b63d194f132f30a/src/views/EditorHome.vue#L515-L523

Also, when a [multi-engine function](https://github.com/VOICEVOX/voicevox_project/issues/2) is released, it can take quite a while to launch multiple engines.
At that time the user may want to open the GUI and uninstall the slower engine.
In that case, the user may have to wait for the splash screen to disappear.
In this context, it may be user-friendly to close the splash screen immediately after the creation of an `AudioCell', rather than after the engine startup is completed (or can choose).

---

コードを見直して気づいたのですが、新しいキャラクターがいた場合、キャラクター選択ダイアログが表示されます。
https://github.com/VOICEVOX/voicevox/blob/c1ae436cb643c78b51653eeb5b63d194f132f30a/src/views/EditorHome.vue#L511-L513

なので、エンジン起動完了より前にスプラッシュ画面を閉じたいです。
この直後にある、`AudioCell`が１つ作成されたタイミングでスプラッシュ画面を閉じるのはどうでしょう。
https://github.com/VOICEVOX/voicevox/blob/c1ae436cb643c78b51653eeb5b63d194f132f30a/src/views/EditorHome.vue#L515-L523

また、[複数エンジン機能](https://github.com/VOICEVOX/voicevox_project/issues/2)がリリースされると、複数のエンジンの起動にかなりの時間がかかる場合があります。
そのときはユーザーはGUIを開いて遅いエンジンをアンインストールしたいかもしれません。
その場合スプラッシュ画面が消えるのを待ち続ける必要がありそうです。
その意味でも、エンジン起動完了後ではなく、いったん`AudioCell`作成直後でスプラッシュ画面を閉じるようにしておくとユーザーフレンドリーかもしれません。